hack/both.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+
```

Comment:
as a matter of principle, scripts should run with `set -e` and errors should be handled (or left unhandled an be fatal).

well, if you insist on handling errors below, then indeed a `set -e` may make it a bit more cumbersome. But such error handling `if [ $rc -eq 0 ]` seems unnecessary anyway. If you want to make the script easier to debug, maybe use `set -ex`.